modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp

```diff
@@ -316,6 +316,8 @@ portletDisplay.setURLBack(backURL);
 					}
 				}
 			});
+
+			loadingElement.remove();
```

Comment:
@icgsbr I'm a little worried about this change because we won't wait anymore for the request response to remove the loading element.

I noticed that we have the following [code](https://github.com/liferay-objects/liferay-portal/blob/1ac48fe2990a6e0404ed724d247e013b8d111f7d/modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp#L139-L140):

```
current.validate().then((result) => {
	if (result) {
       ...
    }
```
and we are only removing the loading element when the `result` is true. 

Could you please check if we add an else statement the issue will be fixed?

```
current.validate().then((result) => {
	if (result) {
       ...
    }
	else {
		loadingElement.remove();
	}
```

Reply:
Done